chcemvediet/apps/inforequests/templates/inforequests/detail/action/content.html

```diff
@@ -7,12 +7,11 @@
  %  -- inforequest: chcemvediet.apps.inforequests.models.Inforequest
```

Comment:
@karol-bujacek Pls rebasni to na aktualny master, nech zbehne pipeline.

--
*) Tento komentar sa netyka riadku, na ktorom je zaveseny. Vesam ho tu len koli tomu, aby sa thread dal resolvnut.